.travis.yml

```diff
@@ -35,7 +35,7 @@ install:
       conda install python=3.4.3;
     fi
   - rm -r $SITE_PACKAGES/numpy*
-  - python setup.py install
```

Comment:
So it skips installing the examples mainly. If you use ``develop`` then the examples are already in a place where it will find them.